src/node_shell_read.js

```diff
@@ -5,12 +5,6 @@
  */
 
 read_ = (filename, binary) => {
-#if SUPPORT_BASE64_EMBEDDING
-  var ret = tryParseAsDataURI(filename);
-  if (ret) {
-    return binary ? ret : ret.toString();
-  }
-#endif
```

Comment:
`read*` can be used for other files than the wasm binary. Does this PR need to add data URI logic to all the possible callers? (if so, does it?)

Reply:
Since all tests pass I think we can say that the answer is no, we don't support arbitrary calls to `read*` with data URLs.  

If we had external callers of this function it seems unlikely/wrong they could be depending on this behavior in these lower level APIs.   The `SUPPORT_BASE64_EMBEDDING` is an internal setting which users cannot directly enable, but which is set only when `SINGLE_FILE` or memory file embedding is enabled.   Its not enabled for general purposes, just for the wasm file and the memory file.. and in both cases its handled at the higher level.